src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement/Program.cs

```diff
@@ -9,6 +12,14 @@
 
 WebApplication app = AccessManagementHost.Create(args);
 
+var appsettings = app.Services.GetRequiredService<IOptions<AccessManagementAppsettings>>().Value;
+if (appsettings.RunInitOnly)
+{
+    using var scope = app.Services.CreateScope();
+    await scope.ServiceProvider.GetRequiredService<AppDbContext>().Database.MigrateAsync();
```

Comment:
Så vi setter opp init bare for EF nå?
og ingen egen feature for EFMigrationDb, så RunInitOnly appsetting blir i praksis toggle for å kjøre EF migrering?

Reply:
vi kan sikkert lage et feature flag om migrerings skal skje eller ikke. Vi må dra denne helt ut i prod før vi kan enable init containerene